request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -41,34 +41,34 @@ import retrofit2.http.Url
 /**
  * Retrofit service for the Vimeo API.
  *
- * TODO: Add documentation to functions.
+ * @see VimeoApiClient
  */
 @Suppress("unused", "UndocumentedPublicFunction", "ComplexInterface")
 interface VimeoService {
 
     @GET("me/connected_apps")
     fun getConnectedAppList(
         @Header(AUTHORIZATION) authorization: String,
+        @QueryMap options: Map<String, @JvmSuppressWildcards String>,
         @Header(CACHE_CONTROL) cacheControl: CacheControl?
     ): VimeoCall<ConnectedAppList>
 
     @GET("me/connected_apps/{type}")
     fun getConnectedApp(
         @Header(AUTHORIZATION) authorization: String,
         @Path("type") type: String,
+        @QueryMap options: Map<String, @JvmSuppressWildcards String>,
```

Comment:
It's because of `in` and `out` variance in Kotlin. In Java, type parameters are often invariant, while in Kotlin they are encouraged to vary if possible for increased utility. For example, here, the `Map` interface in Kotlin has a covariant value type, while in Java's `Map` the value type is invariant. Metaprogramming utilities like Dagger and Retrofit have difficulty handling co- and contravariance in type parameters since it is much harder to validate types when the definition accepts a superset or subset of a type. As a result, these utilities usually throw errors when encountering wildcards, so here we need to suppress the wildcard introduced by Kotlin's `Map` type.

You can read more about the specific issue here: https://github.com/square/retrofit/issues/1805